nike_bot.py

```diff
@@ -1,63 +1,63 @@
-if __name__ == "__main__":
-	import requests
-	import time
-	import twitter
+#if __name__ == "__main__":
+#	import requests
+#	import time
+#	import twitter
 
 	# setup your own account and get these values from twitter
-	api = twitter.Api(consumer_key="CONSUMER_KEY_HERE",
-		consumer_secret="CONSUMER_SECRET_HERE",
-		access_token_key="ACCESS_TOKEN_HERE",
-		access_token_secret="ACCESS_TOKEN_SECRET")
+#	api = twitter.Api(consumer_key="CONSUMER_KEY_HERE",
+#		consumer_secret="CONSUMER_SECRET_HERE",
+#		access_token_key="ACCESS_TOKEN_HERE",
+#		access_token_secret="ACCESS_TOKEN_SECRET")
 
-	t = time.time()
+#	t = time.time()
 
-	while True:
-		try:
-			r = requests.get('https://www.nike.com/launch/')
-			s = r.text.split('div')
+#	while True:
+#		try:
+#			r = requests.get('https://www.nike.com/launch/')
+#			s = r.text.split('div')

```

Comment:
this makes a request to the webpage www.nike.com/launch and parses the html page returned.

line  17 creates a list of div elements from the html page